index.html

```diff
@@ -96,6 +96,140 @@ <h3 class="skill-title">Skills</h3>
         <div class="work">
           <h3>WORK</h3>
         </div>
+        <div class="recent-work">
+          <h1 class="item4">MY RECENT <br /> WORK</h1>
+        </div>
+      </section>
+      <section class="project">
+        <div class="first-project child">
+          <div class="image">
+            <img class="one" src="assets/images/project-one.png" alt="project-image">
+          </div>
+          <h1 class="project-name"> Base Apparel coming soon page </h1>
+          <p class="about-project"> A daily selection of privately 
+            personalized reads; no accounts or sign-ups 
+            required. has been the industry's standard
+             dummy text ever since the 1500s, when an 
+             unknown printer took a standard dummy
+              text.</p>
+          <ul class="tools-list  ">
+            <li class="tools-item right-border">
+              <a href="https://github.com/DanUshindi/Base/blob/main/style.css">css</a>
+            </li>
+            <li class="tools-item right-border ">
+              <a href="https://github.com/DanUshindi/Base/blob/main/index.html">html</a>
+            </li>
+            <li class="lile right-border ">
+              <a href="#">bootstrap</a>
+            </li>
+            <li class="tools-item ">
+              <a href="#">Ruby</a>
+            </li>
+          </ul>
+          <div class="fnl-btn">
+            <button class="btn" disabled>See project</button>
+          </div>
+        </div>
+        <div class="first-project child">
+          <div class="image">
+            <img class="one" src="assets/images/project-one.png" alt="project-image">
```

Comment:
- [ ]  Good work so far by implementing the design similar to the design provided by the Figma template. Make sure that you added all your image illustration as a `background image` and not in the `img` tag. Please, let's rock it